### ISSUE TITLE
Launch Error: Could not launch JNLP file

### ISSUE THREAD

**Issue by evantill** [karakun/OpenWebStart#26]
- OpenWebStart 0.3.1
- Windows 10 on a parallels virtual machine on macos
- JNLP file of a client application running fine in produciton (using oracle webstart).

I could not start the application (class not found at startup).

logs from console:
```
Fri Sep 06 11:15:12 CEST 2019 [ERROR  ] net.sourceforge.jnlp.runtime.JNLPClassLoader: ERROR java.lang.ClassNotFoundException: javax.rmi.PortableRemoteObject  
   at net.sourceforge.jnlp.runtime.JNLPClassLoader.loadClassExt(JNLPClassLoader.java:1715)  
   at net.sourceforge.jnlp.runtime.JNLPClassLoader.loadClass(JNLPClassLoader.java:1515)  
   at com.ctt.gt.tools.connection.ConnectionServer.getHome(ConnectionServer.java:64)  
   at com.ctt.gt.gestionclient.model.proxy.ProxyServerBeansFactory.connectGestionLoaderBean(ProxyServerBeansFactory.java:64)  
   at com.ctt.gt.gestionclient.model.proxy.ProxyServerBeansFactory.initServerBeans(ProxyServerBeansFactory.java:309)  
   at com.ctt.gt.gestionclient.model.proxy.Proxy.start(Proxy.java:42)  
   at com.ctt.gt.gestionclient.model.GestionModel.start(GestionModel.java:62)  
   at com.ctt.gt.gestionclient.controller.GestionController.start(GestionController.java:49)  
   at com.ctt.gt.application.gtApplication.main(gtApplication.java:96)  
   at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)  
   at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)  
   at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)  
   at java.base/java.lang.reflect.Method.invoke(Method.java:566)  
   at net.sourceforge.jnlp.Launcher.launchApplication(Launcher.java:453)  
   at net.sourceforge.jnlp.Launcher.access$400(Launcher.java:70)  
   at net.sourceforge.jnlp.Launcher$TgThread.run(Launcher.java:782)
```

full logs here: https://gist.github.com/evantill/379c6c99c8d0d35c1361e65024d70d57

**Comment by evantill** [comment]
Note: I have extension resources in my jnlp file:

https://gist.github.com/evantill/379c6c99c8d0d35c1361e65024d70d57#file-mymainapp-jnlp

**Comment by AndreasEhret** [comment]
Please could you also show us your JVM Manager settings (ControlPanel>JVM Manager>Settings and what JVM you have activated in the JVM Manager (see https://openwebstart.com/jvm-manager)?

**Comment by evantill** [comment]
<img width="882" alt="Windows_10_Poste_Client_Gcf" src="https://user-images.githubusercontent.com/609288/64420420-8f0ecc80-d09f-11e9-9248-7a923e370890.png">

<img width="879" alt="Windows_10_Poste_Client_Gcf" src="https://user-images.githubusercontent.com/609288/64420474-ad74c800-d09f-11e9-90a3-cb8960041d9f.png">

**Comment by AndreasEhret** [comment]
Thanks, and the settings (sub dialog on the second screen shot when you press the "Settings" button) are still the defaults?

**Comment by evantill** [comment]
<img width="542" alt="Windows_10_Poste_Client_Gcf" src="https://user-images.githubusercontent.com/609288/64423130-09dae600-d0a6-11e9-8b23-3ea0189bb9cd.png">
(yes still the defaults)

**Comment by Pheil** [comment]
My issue may be related to this.  Looks like I have the same settings.

`
ERROR
java.lang.ClassNotFoundException: netscape.javascript.JSException
        at net.sourceforge.jnlp.runtime.JNLPClassLoader.loadClassExt(JNLPClassLoader.java:1715)
        at net.sourceforge.jnlp.runtime.JNLPClassLoader.loadClass(JNLPClassLoader.java:1515)
        at java.lang.Class.getDeclaredConstructors0(Native Method)
        at java.lang.Class.privateGetDeclaredConstructors(Class.java:2671)
        at java.lang.Class.getConstructor0(Class.java:3075)
        at java.lang.Class.newInstance(Class.java:412)
        at net.sourceforge.jnlp.Launcher.createApplet(Launcher.java:642)
        at net.sourceforge.jnlp.Launcher.launchApplet(Launcher.java:532)
        at net.sourceforge.jnlp.Launcher.access$500(Launcher.java:70)
        at net.sourceforge.jnlp.Launcher$TgThread.run(Launcher.java:785)
Caused by: java.lang.NoClassDefFoundError: netscape/javascript/JSException
        at java.lang.Class.getDeclaredConstructors0(Native Method)
        at java.lang.Class.privateGetDeclaredConstructors(Class.java:2671)
        at java.lang.Class.getConstructor0(Class.java:3075)
        at java.lang.Class.newInstance(Class.java:412)
        at net.sourceforge.jnlp.Launcher.createApplet(Launcher.java:642)
        at net.sourceforge.jnlp.Launcher.launchApplet(Launcher.java:532)
        at net.sourceforge.jnlp.Launcher.access$500(Launcher.java:70)
        at net.sourceforge.jnlp.Launcher$TgThread.run(Launcher.java:785)
Caused by: java.lang.ClassNotFoundException: netscape.javascript.JSException
        at net.sourceforge.jnlp.runtime.JNLPClassLoader.loadClass(JNLPClassLoader.java:1557)
        ... 8 more`

**Comment by AndreasEhret** [comment]
I think your case is different, looks like your JNLP tries to launch an Applet. OWS does not provide support for Applets, see also here: https://jaxenter.com/java-web-start-openwebstart-160913.html

**Comment by sclassen** [comment]
It looks like an incompatibility with Java11.
Can you set the `Restrict JVM version range` to 1.8* instead of 1.8+ and give it another try

**Comment by evantill** [comment]
Yes @sclassen problem solved using `j2se` version `1.8*` in JNLP file.

Thank you.

**Comment by sclassen** [comment]
@evantill We fixed the issue with Java11 and have released version 0.3.2
If you want you can give it a try if your (if your application runs on a Java 11)
